### PR TITLE
Story

### PR DESCRIPTION
**Pull request by kimberleyworld**
Kimberleys first instructor review.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -22,27 +23,158 @@ nav {
...
+.textContent p {
+	margin-top: 0;
+	margin-bottom: 0;
+}
+.textContent {
+	display: flex;
+	justify-content: space-between;
+}
+.foot {
+    left: 0;
```
Comment (charliecog) [inline styles.css]: not sure this is doing anything

File: styles.css [hunk 2]
```diff
@@ -22,27 +23,158 @@ nav {
...
+    bottom: 0;
```
Comment (charliecog) [inline styles.css]: not sure this is doing anything

File: styles.css [hunk 3]
```diff
@@ -22,27 +23,158 @@ nav {
...
+    width: 100%;
+    color: white;
+    text-align: center;
+	display: flex;
```
Comment (charliecog) [inline styles.css:105]: indentation here and the line below

**Review by charliecog — COMMENTED** [review]
Really solid build. good work! couple of bits and then will be good to go

General comment:
- remove any unused images eg flowers.png

File: styles.css (hunk 1 above)
Comment (kimberleyworld) [reply styles.css]: Don't think it is! removed it

File: styles.css (hunk 2 above)
Comment (kimberleyworld) [reply styles.css]: removed too

File: styles.css (hunk 3 above)
Comment (kimberleyworld) [reply styles.css:105]: that's strange it isn't indented on mine

**Pull request merged**